recipe/meta.yaml

```
@@ -45,7 +45,11 @@ requirements:
 
 test:
   requires:
-    - python {{ python_min }}    # [use_noarch]
+    # The linter will complain about the missing python_min, but we need
+    # this unpinned to allow cython to be used in the downstream tests
+    # of the python-feedstock.
+    # See also https://github.com/conda-forge/conda-forge.github.io/pull/2387
+    - python                     # [use_noarch]
```

We need to avoid that smithy builds for each python version in the `use_noarch` case

```suggestion
    - python *                   # [use_noarch]
```